lib/state/ui/middleware.ts

```diff
@@ -91,6 +93,10 @@ export const middleware: S.Middleware = (store) => (
       }
       return next(action);
 
+    case 'TAG_TO_TRASH':
+      store.dispatch(showDialog('TRASH-TAG-CONFIRMATION'));
+      return next(action);
```

Comment:
we should not need this. although as we think about it we think, "when we want to trash a tag then open the dialog"  but the dialog reducer can listen and react on its own without a second dispatch. ping me in Slack if you want to talk through this